Add test plan folder structure, template and readme (#99)
Gamify-IT/docs: dev-manuals/test-plans/README.md

# Test plans

> Contains all the test-plans of all our services, and the test plan templates.

Reviewer: ```suggestion
Contains all test plans of all our services, and the test plan template.
```

## Folder structure

> the test plans belong to their corresponding folder. For example, a test plan created for the Moorhuhn frontend:

Reviewer: ```suggestion
The test plans belong to their corresponding folder.  
For example, a test plan created for the Moorhuhn frontend:
```

> ```
> test-plans/minigames/moorhuhn/frontend/plans

Reviewer: ```suggestion
```bash
test-plans/moorhuhn-frontend/plans
```

```

> As soon as a tester has tested a feature according to a test plan, it has to be placed in the `executed` folder

Reviewer: ```suggestion
As soon as a tester has tested a feature according to a test plan, it has to be placed in the `logs` subfolder of that service.
```

## Naming convention

Test plan filenames should have the following structure:

> ```
> name-service-backend/frontend.md
> example:
> highscore-moorhuhn-frontend.md

Reviewer: ````suggestion
```bash
$ACTION-$SERVICE.md
```
So, for example
```bash
highscore-moorhuhn-frontend.md
````

```

> where `name` is the name of the feature the test plan covers

Reviewer: ```suggestion
where `$ACTION` is the name of the feature the test plan covers.
```

> executed tests should be named

Reviewer: ```suggestion
Executed tests should be named
```

> executed tests should be named
> 
> ```
> date-testername-name-service-backend/frontend.md
> example:
> 2022-08-01-MichaelLinder-highscore-moorhuhn-frontend.md

Reviewer: ````suggestion
Test log filename format is
```bash
$DATE-$TESTER-$ACTION-$SERVICE.md
```
So, for example
```bash
2022-08-01-michael_linder-highscore-moorhuhn-frontend.md
````
